test/Sets/Polytope.jl

```diff
@@ -228,6 +228,15 @@ for N in [Float64, Rational{Int}, Float32]
     P = VPolytope([N[1, 2, 3], N[4, 5, 6]])
     Q = VPolytope([N[3, 1, 2], N[6, 4, 5]])
     @test permute(P, [3, 1, 2]) == Q
+
+    # construction with static arrays
+    vlist = [SVector{2}(N[0, 0]), SVector{2}(N[1, 0]), SVector{2}(N[0, 1])]
+    V = VPolytope(vlist)
+    @test vertices_list(V) == vlist
+    # check that the outer container can also be static
+    vlist = SVector{3}(SVector{2}(N[0, 0]), SVector{2}(N[1, 0]), SVector{2}(N[0, 1]))
+    V = VPolytope(vlist)
+    @test vertices_list(V) == vlist
```

Comment:
```suggestion
    @test ispermutation(vertices_list(V), vlist)
    # check that the outer container can also be static
    vlist = SVector{3}(SVector{2}(N[0, 0]), SVector{2}(N[1, 0]), SVector{2}(N[0, 1]))
    V = VPolytope(vlist)
    @test ispermutation(vertices_list(V), vlist)
```
I think this is "more correctly" what the specification says.
(If you wanted to test that the output is an `SVector`, you have to test that explicitly because I think `==` does not care.)